pennylane/interfaces/tf.py

```diff
@@ -105,6 +103,8 @@ def __repr__(self):
         metric_tensor = qnode.metric_tensor
         draw = qnode.draw
         func = qnode.func
+        set_trainable_args = qnode.set_trainable_args
+        get_trainable_args = qnode.get_trainable_args
```

Comment:
I think the class `qnode_str` might need a better name. I tried to understand why it was important to set these methods here, but then I realized I didn't even understand the purpose of the class itself. I'm guessing it's to turn the `_TFQNode` function into a class?

Reply:
Have updated both instances of `qnode_str` to `TorchQNode` and `TFQNode`, and left an explanation in the code as a comment block.